src/table/table.scss

```diff
@@ -177,22 +186,35 @@
   .iui-row {
     min-width: 100%;
     display: flex;
-    border: solid 1px transparent;
-    @media (prefers-reduced-motion: no-preference) {
-      transition: border $iui-speed-fast ease-out;
-    }
-    @include themed {
-      border-bottom-color: t(iui-color-background-border);
+
+    &:not(.iui-expanded-content) .iui-cell {
+      border-top: solid 1px transparent;
```

Comment:
Shouldn't all these go in the `iui-table-cell` mixin?

Reply:
Also I feel like we should be able to apply borders to cell for `iui-expanded-content` too, because it's just one big cell.

Reply:
`iui-table-cell` mixin is reused for header and body and I only need these rules for body cells.
Only in HTML example we are demoing expanded content that has `iui-cell` inside but in React we are not using it as it can be anything so I don't know how can I add borders to expanded content only when there is no `iui-cell` except for using `:has` which is not fully supported yet. Because otherwise I will get double borders.

Reply:
> `iui-table-cell` mixin is reused for header and body and I only need these rules for body cells.

`@at-root iui-table-body iui-row:not(.iui-expanded-content) &` should work, no?

> Only in HTML example we are demoing expanded content that has `iui-cell` inside but in React we are not using it

Can we remove iui-cell from the html example then? Looks confusing.

Reply:
> `@at-root iui-table-body iui-row:not(.iui-expanded-content) &` should work, no?

How it is better? Also we have a lot of `.iui-cell` rules that are wrapped with other classes so then we should be consistent and move those to mixin too. And I don't want to do that, too much changes then.

> Can we remove iui-cell from the html example then? Looks confusing.

I think we can. It doesn't feel right to use `iui-cell` in random place. Maybe I should also remove those `:not(.iui-expanded-content)`?

Reply:
Removed `iui-cell` from expanded contents